CODE_OF_CONDUCT.md

```diff
@@ -21,28 +21,33 @@
 
 ## 1. Introduction
 
-The aim of this Code of Conduct is to empower all participants at
-Quansight-organized in-person events in Quansight-managed public repositories
-to facilitate healthy and constructive community behavior. Those participants
+The aim of this Code of Conduct is to facilitate healthy and constructive
+community behavior by all participants at
+Quansight-organized in-person events and in Quansight-managed public
+repositories.  Those participants
 may be users, maintainers, or contributors to open-source projects managed by
 Quansight LLC or Quansight Labs. This Code of Conduct outlines expected
 behaviors as well as steps for reporting incidents of unacceptable behavior.
 
 This Code of Conduct applies to all repositories and communities for
 Quansight-managed open-source projects regardless of whether or not the
 repository explicitly includes this document. The Code also applies in public
-spaces when an individual represents a project or its community. Examples
+spaces when an individual represents a project or its community. This Code of
+Conduct applies to all spaces and public repositories of Quansight on all
+platforms (including Github). Examples
 include using an official project e-mail address, posting via an official
 social media account, or acting as an appointed representative at an online or
 in-person event. What constitutes representing a project may be further defined
 and clarified by project maintainers.
 
 Quansight is committed to providing a welcoming and inspiring community for
 all. Anyone claiming affiliation with Quansight public repositories is required
-to conform to the Code of Conduct. This Code of Conduct applies to all spaces
-and public repositories of Quansight on all platforms (including Github).
+to conform to the Code of Conduct. 
 Possible consequences for breaching this Code of Conduct are listed below.
 
+Incidents concerning Quansight employees may be refered to the Quansight HR
+staff.
+
```

Comment:
Is this covered in some other company-wide agreement that would make this sentence redundant?

Reply:
This works adequately. Either the Employee Handbook or the company Code of Conduct contains relevant wording about reporting to HR. This document, however, likely needs this line as it's accessible to a broader group. It's also useful to keep this self-contained.